dart/lib/src/client.dart

```diff
@@ -188,15 +109,63 @@ abstract class SentryClient {
     options.httpClient?.close();
   }
 
-  @override
-  String toString() => '$SentryClient("$postUri")';
+  SentryEvent _processEvent(
+    SentryEvent event, {
+    dynamic hint,
+    List<EventProcessor> eventProcessors,
+  }) {
+    for (final processor in eventProcessors) {
+      event = processor(event, hint);
```

Comment:
if one of the processors returned null, means that this is event is dropped, so we should bail out and log it.
eg https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/SentryClient.java#L155-L184

Reply:
I can help with those things on a different PR though

Reply:
✅  fixed